### DISCUSSION TITLE
Table vertical lines missing

### DISCUSSION THREAD

**Discussion by ronwildt** [squidfunk/mkdocs-material#3952] [Q&A] [answered]
Mkdocs table vertical lines are missing...potential bug.  Don't see a way to make them show up as in standard markdown?

Typical mkdocs table:

![image](https://user-images.githubusercontent.com/29985566/170318177-03493b95-c04c-4be3-8c59-79b89170f346.png)

Should show up like below...how to rectify?

![image](https://user-images.githubusercontent.com/29985566/170318292-636f23a7-1113-41c2-98b7-51e84754f63f.png)

**Comment by facelessuser** [comment, marked as answer]
They aren't missing, that is a stylistic choice for Material. If you want verticle lines, you'll have to provide additional CSS to do so.

**Comment by ronwildt** [comment]
Isaac,

Thanks unfortunate as having standard lines for tables would be much more helpful.  Do you happen to have an example of how this could be done or can you point me in the right direction?

I found this link but it doesn’t appear to work properly and no directions.

I appreciate any and all assistance.

Have a great day & a terrific weekend!
Thanks,
Ron

From: Isaac Muse ***@***.***>
Reply-To: squidfunk/mkdocs-material ***@***.***>
Date: Wednesday, May 25, 2022 at 12:46 PM
To: squidfunk/mkdocs-material ***@***.***>
Cc: "Ron.Wildt" ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [squidfunk/mkdocs-material] Table vertical lines missing (Discussion #3952)


They aren't missing, that is a stylistic choice for Material. If you want verticle lines, you'll have to provide additional CSS to do so.

**Comment by ronwildt** [comment]
Sorry, forgot the link

https://blog.ktz.me/making-mkdocs-tables-look-like-github-markdown-tables/

Have a great day!
Thanks,
Ron

From: "Ron.Wildt" ***@***.***>
Date: Thursday, May 26, 2022 at 1:40 PM
To: squidfunk/mkdocs-material ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [EXTERNAL] Re: [squidfunk/mkdocs-material] Table vertical lines missing (Discussion #3952)

Isaac,

Thanks unfortunate as having standard lines for tables would be much more helpful.  Do you happen to have an example of how this could be done or can you point me in the right direction?

I found this link but it doesn’t appear to work properly and no directions.

I appreciate any and all assistance.

Have a great day & a terrific weekend!
Thanks,
Ron

From: Isaac Muse ***@***.***>
Reply-To: squidfunk/mkdocs-material ***@***.***>
Date: Wednesday, May 25, 2022 at 12:46 PM
To: squidfunk/mkdocs-material ***@***.***>
Cc: "Ron.Wildt" ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [squidfunk/mkdocs-material] Table vertical lines missing (Discussion #3952)


They aren't missing, that is a stylistic choice for Material. If you want verticle lines, you'll have to provide additional CSS to do so.

**Reply by facelessuser** [reply]
I'm not going to go into detail on how to style like GitHub, but you should be able to get vertical lines with something like what is shown below. Just replace `magenta` with the appropriate color:

```css
.md-typeset td:not([class]):not(:last-child) {
    border-right: 1px solid magenta;
}
```

**Reply by anushibinj** [reply]
I modified this a bit more and used the following CSS to show all borders:

```css
.md-typeset td:not([class]):not(:last-child) {
	border: 0.05rem solid var(--md-typeset-table-color);
}
```

**Comment by ronwildt** [comment]
I tried the code below, as you can see below in mkdocs.yml but it didn’t make any difference to pages with a table in them, example below after rebuilding the site locally, fyi.  Any additional thoughts?  Thanks. 😊

  extra_css:
    - stylesheets/extra.css

***@***.***


Have a great day!
Thanks,
Ron

From: Isaac Muse ***@***.***>
Reply-To: squidfunk/mkdocs-material ***@***.***>
Date: Thursday, May 26, 2022 at 1:51 PM
To: squidfunk/mkdocs-material ***@***.***>
Cc: "Ron.Wildt" ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [squidfunk/mkdocs-material] Table vertical lines missing (Discussion #3952)


I'm not going to go into detail on how to style like GitHub, but you should be able to get vertical lines with something like what is shown below. Just replace magenta with the appropriate color:

.md-typeset td:not([class]):not(:last-child) {

    border-right: 1px solid magenta;

}

**Reply by facelessuser** [reply]
I don't know what to say. Doing this to the official Material docs, it works fine:

<img width="955" alt="Screen Shot 2022-05-26 at 1 15 53 PM" src="https://user-images.githubusercontent.com/1055125/170560770-be3e92f4-ddfb-493a-a19e-d37c46b4fe18.png">
